src/vivarium_public_health/disease/state.py

```diff
@@ -446,6 +446,9 @@ def __init__(
             excess_mortality_rate
         )
 
+        self.dwell_time_pipeline_name = f"{self.state_id}.dwell_time"
```

Comment:
Same as in the vivarium PR - should we call these something like `self.foo_pipeline` instead of `self.foo_pipeline_name`?